phone/src/apps/match/hooks/useMatchNotifications.ts

```diff
@@ -38,9 +37,10 @@ export const useMatchNotifications = () => {
       notificationIcon,
     };
 
-    addNotificationAlert(notification)
+    addNotificationAlert(notification);
     addNotification(notification);
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
Please don't ignore eslint warnings. Let's try and fix it instead.

Here you should add both `addNotificationAlert` and `addNotification` to the dependency array.

Reply:
You think that I should also add icon and notificationIcon?

Reply:
Yeah :)

Reply:
Yeah